src/main/scala/is/hail/keytable/KeyTable.scala

```diff
@@ -799,9 +807,10 @@ case class KeyTable(hc: HailContext, rdd: RDD[Row],
     val maybeTieBreaker = tieBreakerExpr.map { e =>
       val tieBreakerThunk = Parser.parseTypedExpr[Int](e, tieBreakerEc)
 
-      { (l:Any, r: Any) =>
-        tieBreakerEc.setAll(l, r)
-        tieBreakerThunk() }
+    { (l: Any, r: Any) =>
+      tieBreakerEc.setAll(l, r)
+      tieBreakerThunk()
+    }
```

Comment:
I'm confused by the indentation change. It looked right to me before.

Reply:
Huh, it looks like IntelliJ was mis-indenting.  I modified the code slightly and now it indents correctly.